src/directives/onLongPress.ts

```diff
@@ -0,0 +1,118 @@
+import {ElementPart, nothing} from 'lit';
+import {
+	Directive,
+	directive,
+	DirectiveClass,
+	DirectiveResult,
+	PartInfo,
+	PartType,
+} from 'lit/directive.js';
+
+class LongPressDirective extends Directive {
+	__element!: Element;
```

Comment:
I used `#` to make everything private instead, is it a good idea? I don't see why we would need to augment this directive so I don't see `protected` keyboard fitting here.
Do you like this modification?

Reply:
i _think_ it should be ok as long as the typescript emit doesn't emit the private fields (and rather downlevels it). otherwise we'd be bumping the version of js we support